### PR TITLE
[#168821528] Add ValidateProfileEmail function

### PR DESCRIPTION
**Pull request by francescopersico**

### CODE REVIEW

File: ValidateProfileEmail/index.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+
+import { ValidateProfileEmail } from "./handler";
+
+// Setup Express
+const app = express();
+secureExpressApp(app);
+
+const storageConnectionString = getRequiredStringEnv("StorageConnection");
+const cosmosDbUri = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_URI");
```
Comment (gunzip) [inline ValidateProfileEmail/index.ts]: connection strings ?

File: package.json [hunk 2]
```diff
@@ -40,10 +40,10 @@
     "durable-functions": "^1.2.4",
     "express": "^4.15.3",
     "fp-ts": "1.12.0",
-    "io-functions-commons": "^1.5.0",
+    "io-functions-commons": "1.6.0",
```
Comment (gunzip) [inline package.json]: you may want to upgrade to 3.x

Comment (francescopersico) [reply package.json]: 3.x or 4.x?

**Comment by gunzip** [comment]
4.x sorry :)


Il giorno sab 14 mar 2020 alle ore 11:19 Francesco Persico <
notifications@github.com> ha scritto:

**Comment by francescopersico** [comment]
All done

**Pull request merged**